index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav>
+      <h2>Welcome</h2>
+      <img src="./images/menu.svg" alt="menu-icon" />
+    </nav>
+    <main>
+      <h1>
+        Hey there. I’m Diego <br />
+        <span>I’m a software developer</span>
+      </h1>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <img src="./images/twitter.svg" alt="twitter" />
+        <img src="./images/linkedin.svg" alt="linkedin" />
+        <img src="./images/medium.svg" alt="medium" />
+        <img src="./images/github.svg" alt="github" />
+        <img src="./images/angellist.svg" alt="angellist" />
+      </div>
+      <img src="./images/arrowdown.svg" alt="arrowdown" />
```

Comment:
- [ ] Kindly be noticed that you should wrap all of that section inside a `section` tag.
- [ ] Also be noticed that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).